pkg/server/sandbox_status.go

```diff
@@ -132,10 +112,23 @@ type sandboxInfo struct {
 }
 
 // toCRISandboxInfo converts internal container object information to CRI sandbox status response info map.
-func toCRISandboxInfo(ctx context.Context, sandbox sandboxstore.Sandbox,
-	pid uint32, processStatus containerd.ProcessStatus, verbose bool) (map[string]string, error) {
-	if !verbose {
-		return nil, nil
+func toCRISandboxInfo(ctx context.Context, sandbox sandboxstore.Sandbox) (map[string]string, error) {
+	container := sandbox.Container
+	task, err := container.Task(ctx, nil)
+	if err != nil && !errdefs.IsNotFound(err) {
+		return nil, fmt.Errorf("failed to get sandbox container task: %v", err)
+	}
+
+	var pid uint32
+	var processStatus containerd.ProcessStatus
+	if task != nil {
+		taskStatus, err := task.Status(ctx)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get task status: %v", err)
+		}
+
+		pid = task.Pid()
+		processStatus = taskStatus.Status
```

Comment:
Could we also store the `pid` and `processStatus` to `sandboxstore`?

Reply:
`processState` keeps changing, it's hard to maintain it in our store. I'd prefer not to do that until we have to.

I'll do `pid`.

Reply:
Done